### DISCUSSION TITLE
Wrangler command does not work even after installing it

### DISCUSSION THREAD

**Discussion by Dhruvacube** [community/community#17108] [Codespaces]
### The [wrangler](https://github.com/cloudflare/wrangler) cli by cloudflare dosent work in codespace even after installing it :(

![image](https://user-images.githubusercontent.com/34207400/169696304-602a3bb5-672c-484c-a0d3-dc2d9501b0a6.png)

**Comment by mrbm** [comment]
Have you had any success with wrangler 2?

**Reply by Dhruvacube** [reply]
never tried it yet

**Reply by Dhruvacube** [reply]
lemme try it
